util/src/main/scala/outwatch/util/Http.scala

```diff
@@ -70,7 +70,7 @@ object Http {
     headers = request.headers,
     withCredentials = request.withCredentials,
     responseType = request.responseType
-  )
+  ): @scala.annotation.nowarn("cat=deprecation") // TODO
```

Comment:
The AJAX helpers in scala-js-dom are deprecated. It'll take a bit of work to rework it here, ok to leave to a future PR?

Reply:
I think, that is fine for now :+1: